### PR TITLE
users can vote in an election #16

### PR DESCRIPTION
**Pull request by danieliheonu**

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ class Vote(db.Model):
     password = db.Column(db.String())
 
     def __repr__(self):
-        return f'<Vote {self.id}>'
+        return f'<Vote {self.election} {self.candidate} {self.user}>'
```
Comment (LeanKhan) [inline app/models.py:82]: I'm not sure how this works though. Are you displaying the entire Election, Candidate, or User objects

Comment (danieldutcum) [reply app/models.py:82]: It does. It's basically how it'll appear if your query the Vote table

Comment (danieliheonu) [reply app/models.py:82]: Before it was showing only the id and I didn't like it cos it wasn't self explanatory so I changed it to show the name of the election, the candidate (you're voting for) and the user that's voting

**Review by LeanKhan — COMMENTED** [review]
Looks good, will see how we can merge without conflicts from my own update-election branch

**Review by LeanKhan — APPROVED** [review]

**Pull request merged**